content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly name service ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** container ports are no longer required in the pod spec. All ports will be captured by default.
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](docs/ops/traffic-management/locality-load-balancing/) by default.
+- **Improved** Pilot performance. The reduction in CPU utilization could be up to 10 times depending on deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** metrics to the Citadel agent to monitor SDS push.
+- **Improved** the ingress gateway to provide client CA certificates when using SDS.
+- **Added** trust domain validation for Envoy proxies using mutual TLS. Improved validation only applies to the `Sidecars` resource.
+- **Added** the `ca.istio.io/env` and `ca.istio.io/override` labels for operators to control service account secret generation by namespace.
+- **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration a future release.
+- **Added** SDS support to improve the performance of Envoy proxies handling secrets.
```

Comment:
this is confusing.  SDS has been added at least in 1.2 or earlier.

Reply:
+1, I think this was a misinterpretation of the original feature. Not sure what that feature actually is though

Reply:
Perhaps this should be **Promoted** instead?

Reply:
I think the confusion here is that we have modified this row so many times that it no longer describing what was actually changed. This came from

> Updated SDS feature to be enabled for the Istio control plane.

Right? The change was that SDS is enabled for the control plane (Pilot, Mixer, etc). It has nothing to do with "performance of Envoy proxies handling secrets"

Reply:
John is right. We added control plane SDS in 1.3. @JimmyCYJ

Reply:
@linsun @howardjohn @myidpt @JimmyCYJ Would this suggestion address everyone's concerns?
```suggestion
- **Enabled** SDS for the control plane to improve the performance of Envoy proxies handling secrets.
```

Reply:
I revised it. Please take a look.

Reply:
I wouldn't say "enabled" because SDS is not enabled by default (It's an Alpha feature). Maybe just use what I suggested down below? :)